analysers/trials/executionTime.py

```diff
@@ -15,28 +15,26 @@
 
 def computeExecutionTime(dataRDD):
     if(dataRDD.isEmpty()):
-        return (None, None)
+        return None
     
     data = dataRDD.map(lambda r: (r['start_time'], r['end_time'])) \
             .collect()
     
     smallest = None
-    for d in data:
-        t = d[0]
-        if smallest == None:
-            smallest = t
-        elif t != None:
-            if t < smallest:
-                smallest = t
-            
     largest = None
     for d in data:
```

Comment:
Add comments in the following code so that I can understand better what is going on. Moreover, assign the first value of the data array to both `largest` and `smallest` and then iterate the data array from the second element to update the min and max.